### PR TITLE
Update Text File Section

### PR DESCRIPTION
**Pull request by brishtibheja**
A little polishing and I removed some outdated info I thought isn't necessary anymore.

### CODE REVIEW

File: src/importing/text-files.md [hunk 1]
```diff
@@ -155,13 +151,6 @@ so if you have a file named apple.jpg, the front would say 'apple') and
 the images or audio on the back. If you would like a different
 arrangement of media and filenames, you can [change the note type](../browsing.md) of the created cards afterwards.
 
-## Adding Tags
-
-If you want to add 'tag1' and 'tag2' to every line you’re importing, add
-the following to the top of the text file:
-
-    tags:tag1 tag2
-
```
Comment (dae) [inline src/importing/text-files.md:164]: This is distinct from the tags: entry below - note how the first line doesn't need a comment. But I'd prefer we push people to use the more consistent approach mentioned below, so I think it's ok to remove this.

Comment (brishtibheja) [reply src/importing/text-files.md:164]: Yeah, that's what I thought too.

File: src/importing/text-files.md [hunk 2]
```diff
@@ -12,10 +12,6 @@ are met.
 
 - The files must be in UTF-8 format (see below).
 
-- The first line also defines the separating character – if Anki finds
```
[the remark this answers is not in the document]
Comment (brishtibheja) [reply src/importing/text-files.md:15]: But as people can change it, does it really needs documented?

[the remark this answers is not in the document]
Comment (dae) [reply src/importing/text-files.md:15]: Ok, it could be mentioned in the separator section of the table below instead for example.

**Pull request merged**